locales/de.js

```diff
@@ -30,5 +30,5 @@
   "Create invite with `%s` command": "Erstelle eine Einladung mit dem Befehl `%s`",
   "Command failed": "Der Befehl war erfolglos",
   "DISBOARD API server seems to be down": "Der DISBOARD API Server scheint außer Betrieb zu sein",
-  "You are rate limited. Please wait %s seconds": "You are rate limited. Please wait %s seconds"
+  "You are rate limited. Please wait %s seconds": "Deine Anfragen werden begrenzt. Bitte warte %s Sekunden"
```

Comment:
I spammed reactions on Discord until I got the notification — their official translation of "You are rate-limited" would be "**Deine Aktionsrate wird begrenzt**" in case you want to use that?

Reply:
How about "Deine Aktionsrate wird limitiert"?

Reply:
I don't think it makes any difference, so you can choose what you favour :)